org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/client/IRepl.java

```diff
@@ -22,13 +22,15 @@
      *
      * @param input
      *            The input to send for evaluation.
-     * @return An {@link IHook} to process the result of the evaluation.
+     * @return An {@link IResult} to process the result of the evaluation, or {@code null} when an
```

Comment:
This javadoc is outdated: an "empty" `IResult` is returned when the input is empty.